### ISSUE TITLE
Gear ~ unexpected ui of Play toolbar icon.

### ISSUE THREAD

**Issue by hharshas** [llaske/sugarizer#1600]
After pressing play toolbar icon, when we press other toolbar icons, the play action stops ,but the play svg on the play toolbar icon remains instead of switching to the pause svg.
![Screenshot from 2024-04-18 21-51-09](https://github.com/llaske/sugarizer/assets/125907752/a40b0a4f-a2aa-4598-b7ef-398c645ae963)

**Comment by llaske** [comment]
You're right, it's related to https://github.com/llaske/sugarizer/pull/1506

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/1601
